internal/testplanet/planet.go

```diff
@@ -428,7 +429,6 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 
 				Config: tlsopts.Config{
 					RevocationDBURL:     "bolt://" + filepath.Join(storageDir, "revocation.db"),
-					UsePeerCAWhitelist:  true,
```

Comment:
This should be still here.

Reply:
no, it's being deleted here

Reply:
I meant that this PR is removing the whitelist, but this would disable the checking everywhere. Since the run planet already reconfigures things to disable it, I'm not understanding what its benefit is.

Reply:
Sorry, I got ahead of myself. I wasn't paying attention to where this was. Yea, this probably shouldn't be removed.